### ISSUE TITLE
My Azure fucntion project is on .Net 4.7.1. Running into issues with the system dlls.

### ISSUE THREAD

**Issue by NNara** [Azure/Azure-Functions#722]
![image](https://user-images.githubusercontent.com/36491202/37614889-1c3e8efc-2b69-11e8-9e6e-5da4cd0e4faf.png)

How do I resolve the compilation issues? I tried adding .NetStandard 2.0.0 nugget and it doesn't fix the problem. however, if I downgrade my Azure functions project to .Netframework 4.6.1, we do not run into this compilation issue.

Can you confirm if the latest .net framework version is not supported in Azure functions? If yes, what are the resolution steps to move forward with above issue.

Appreciate your help!!

**Comment by pragnagopa** [comment]
Can you give us more details to repro this issue? Are you trying to use Microsoft.ServiceFabric.Actors nuger from within the functionapp?  Function App ->V1 or V2

**Comment by NNara** [comment]
Yes. We are using the nugget within funcitonapp. How do I know the version of Azure function?

**Comment by fabiocav** [comment]
@NNara apologies for the delayed response; some of the issues here have been missed. Are you still running into problems?
